lib/rubocop/cop/rspec/focus.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if test is focused.
+      #
+      # @example
+      #   # bad
+      #   describe MyClass, focus: true do
+      #   end
+      #
+      #   # good
+      #   describe MyClass do
+      #   end
+      class Focus < Cop
+
+        MESSAGE = "Focused spec found.".freeze
+
+        FOCUSABLE_BLOCKS = [
+          :describe, :context, :it, :feature, :scenario
```

Comment:
What about `specify`? Also, the pending variants such as `xit`, `xdescribe` etc.